### PR TITLE
A more detailed summary of performance runs available with 'get performance' command

### PR DESCRIPTION
**Pull request by benjaminstew**
- changed scenario_performance_map to include multiple performance metrics from mlperf_log_summary.txt file 
- simplified scenario_performance_map (the beautify_summary function in code_axs already does all the formatting of units etc.. required) 
- added 99p latency: target latency ratio to 'get performance' summary for server runs

### CODE REVIEW

**Comment by bmsgit1** [comment]
sorry @xihajun I accidentally removed saheli !!

**Review by kaidrake — COMMENTED** [review]
I have tested the changes on my experiment entries for classic benchmarks for Offline and Server. It looks great:

```
eselezne@aus121-r760-0:/local/mnt/workspace/eselezne/work_collection/experiments$ axs byname generated_by_image_classification_using_qaic_kilt_on_get_8cd307069273465a828fc88f09904754 , get performance
['VALID :', 'Samples_per_second=425210', 'target_qps=435000']

eselezne@aus121-r760-0:/local/mnt/workspace/eselezne/work_collection/experiments$ axs byname generated_by_bert_using_qaic_kilt_on_get_d649d2abe5bc43b7a240ab255830fc4a , get performance
['VALID :', 'Samples_per_second=14821.5', 'target_qps=14600']

eselezne@aus121-r760-0:/local/mnt/workspace/eselezne/work_collection/axs2mlperf$ axs byname generated_by_object_detection_using_qaic_kilt_on_get_2b473f54a8fe45e395d714b4cf6eab62 , get performance
['VALID :', 'Samples_per_second=7081.97', 'target_qps=7500'] 
```

```
eselezne@aus121-r760-0:/local/mnt/workspace/eselezne/work_collection/experiments$ axs byname generated_by_image_classification_using_qaic_kilt_on_get_cc564210f6fa4999a07d78e78eb2071b , get performance
['VALID :', 'target_qps=426600', '99.00_percentile_latency=13.664 milliseconds', 'target_latency=15.000 milliseconds', 'latency_cutoff_ratio=0.9109276666666667', 'Completed_samples_per_second=426699.58']

eselezne@aus121-r760-0:/local/mnt/workspace/eselezne/work_collection/experiments$ axs byname generated_by_bert_using_qaic_kilt_on_get_de793ffb35d04cc8aa22e6bdc48fb8f0 , get performance
['VALID :', 'target_qps=6850', '99.00_percentile_latency=87.096 milliseconds', 'target_latency=130.000 milliseconds', 'latency_cutoff_ratio=0.6699658615384615', 'Completed_samples_per_second=6851.47']

eselezne@aus121-r760-0:/local/mnt/workspace/eselezne/work_collection/experiments$ axs byname generated_by_object_detection_using_qaic_kilt_on_get_bb1bbdcd0ed94e289ab8be7b3c78960e , get performance
['VALID :', 'target_qps=6350', '99.00_percentile_latency=86.401 milliseconds', 'target_latency=100.000 milliseconds', 'latency_cutoff_ratio=0.86400786', 'Completed_samples_per_second=6351.55']
```

With respect to the code, I spotted some extra spaces: [base_loadgen_experiment/code_axs.py](https://github.com/krai/axs2mlperf/pull/54/commits/1551d272e4c52fab874f1b4139d22e5ff121e498#diff-f85c41262c69079dcd709e13d7f5bebd61e49c32feeef6a15cbc9d552ed16685)
L49,L82
[base_loadgen_experiment/data_axs.json](https://github.com/krai/axs2mlperf/pull/54/commits/1551d272e4c52fab874f1b4139d22e5ff121e498#diff-df7bd58196a07ab4846374a5bc0c250c4ac12ceb5d2dba7476743d08d794ca50)
L37

**Comment by bmsgit1** [comment]
yep I will remove those spaces and the unsightly colon after the validity!

File: base_loadgen_experiment/code_axs.py [hunk 1]
```diff
@@ -61,21 +65,40 @@ def parse_summary(abs_log_summary_path):
     return beautified_summary
 
 
-def parse_performance(summary, scenario_performance_map, raw=False):
+def calc_latency_cutoff_ratio(parsed_summary):
+    
+    scenario = parsed_summary["Scenario"]
+
+    if scenario == "Server":
+        return parsed_summary["99.00_percentile_latency_ns"]/parsed_summary["target_latency_ns"]
```
Comment (sahelib25) [inline base_loadgen_experiment/code_axs.py:74]: Can you make `latency_cutoff_ratio` rounded off to 2 decimal places?

Comment (bmsgit1) [reply base_loadgen_experiment/code_axs.py:74]: sure!

**Comment by bmsgit1** [comment]
The command ouput now looks like this: ['VALID', 'target_qps=152000', '99.00_percentile_latency=7.722 milliseconds', 'target_latency=15.000 milliseconds', 'latency_cutoff_ratio=0.51', 'Completed_samples_per_second=150593.59']

**Pull request merged**